docker-compose.yml

```diff
@@ -478,7 +478,7 @@ services:
             timeout: 10s
             retries: 60
     chainlink-external-adapter:
-        container_name: streamr-dev-chainlink-adapter
+        container_name: streamr-dev-chainlink-external-adapter
```

Comment:
this breaks the chainlink workflow because it is linked directly to the container, instead of 10.200.10.1. I created a ticket to fix this: https://linear.app/streamr/issue/ETH-155